pkg/storage/etcd3/store.go

```diff
@@ -57,6 +57,10 @@ type objState struct {
 	data []byte
 }
 
+func NewStore(c *clientv3.Client, codec runtime.Codec, prefix string) storage.Interface {
```

Comment:
Just New() for a name.

Also the return type can be *store-- people can store that in a storage.Interface. You can document this in a comment.

Actually, maybe just promote the existing newStore to be "New".